src/guides/v2.3/frontend-dev-guide/layouts/xml-manage.md

```diff
@@ -484,9 +484,23 @@ class Class implements \Magento\Framework\View\Element\Block\ArgumentInterface
   {
 
   }
+
+  public function canShowAdditionalData() {
```

Comment:
Please, keep in mind PSR. It happens to me as well when working with javascript too much 😎